core/storage/statistics/gae_models.py

```diff
@@ -824,6 +824,27 @@ def get_multi_versions(cls, exp_id, version_numbers):
         exploration_stats_models = cls.get_multi(entity_ids)
         return exploration_stats_models
 
+    @classmethod
+    def get_multi_stats_models(cls, exp_version_reference_dicts):
+        """Gets stats model instances for each exploration and the corresponding
+        version number.
+
+        Args:
+            exp_version_reference_dicts: list(dict). List of dicts where each
```

Comment:
Use the domain objects here, not their dict representations.

Reply:
Done.